modin/pandas/groupby.py

```diff
@@ -962,16 +962,8 @@ def size(self):
                 idx_name=self._idx_name,
                 **self._kwargs,
             ).size()
-        work_object = type(self)(
-            self._df,
-            self._by,
-            self._axis,
-            drop=False,
```

Comment:
explicitly setting `drop=False` seems useless for the result of `groupby.size()`